.github/workflows/mkdocs.yml

```
@@ -55,31 +58,34 @@ jobs:
           enable-commit-comment: false
           enable-commit-status: false
           overwrites-pull-request-comment: false
-          alias: ${{ steps.netlify-alias.outputs.alias }}
+          alias: ${{ steps.branch.outputs.name }}
           github-deployment-environment: Netlify
           github-deployment-description: Branch deployment preview
         env:
           NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_PERSONAL_ACCESS_TOKEN }}
           NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
 
-      # On push to master, deploy the docs to GitHub Pages
-      - name: Set up git author
+      # Deploy docs on push to master
+      - name: Set up environment variables
         if: github.ref == 'refs/heads/master'
         run: |
-          remote_repo="https://${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
-          git config --global user.name "${GITHUB_ACTOR}"
-          git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
-          git remote rm origin
-          git remote add origin "${remote_repo}"
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          echo "MKDOCS_VERSION=$(mkdocs --version | cut -d " " -f 3)" >> $GITHUB_ENV
+          echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-9`" >> $GITHUB_ENV
+          echo "CUSTOM_DOMAIN=docs.pulse.codacy.com" >> $GITHUB_ENV
 
-      - name: Deploy docs
+      - name: Create robots.txt
         if: github.ref == 'refs/heads/master'
         run: |
-          echo "${CUSTOM_DOMAIN}" > "${GITHUB_WORKSPACE}/docs/CNAME"
-          echo -e "User-agent: *\nSitemap: https://${CUSTOM_DOMAIN}/sitemap.xml" > "${GITHUB_WORKSPACE}/docs/robots.txt"
-          git fetch origin gh-pages --verbose
-          mkdocs -v gh-deploy --config-file "${GITHUB_WORKSPACE}/mkdocs.yml"
-        env:
-          CUSTOM_DOMAIN: docs.pulse.codacy.com
+          echo -e "User-agent: *\nSitemap: https://${{ env.CUSTOM_DOMAIN }}/sitemap.xml" > "${GITHUB_WORKSPACE}/docs/robots.txt"
```

@claudiacarpinteiro, I already tested the changes on codacy/docs and we actually need an adjustment here. :nerd_face:  See https://github.com/codacy/docs/pull/1580 for more information.

```suggestion
          echo -e "User-agent: *\nSitemap: https://${{ env.CUSTOM_DOMAIN }}/sitemap.xml" > "./site/robots.txt"
```